lib/json-schema/schema/reader.rb

```diff
@@ -71,8 +70,6 @@ def read(location)
         JSON::Schema.new(JSON::Validator.parse(body), uri)
       end
 
-      # @param uri [Addressable::URI]
-      # @return [Boolean]
```

Comment:
Because in 6967d99 I've tried to abstract away the fact that we use `Addressable::URI`, however, I'm probably kidding myself. So I've restored this until I have time to make a more water-tight solution.